pkg/operator/starter.go

```diff
@@ -280,14 +280,27 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		return err
 	}
 
+	apiServerConfig, err := configClient.ConfigV1().APIServers().Get(ctx, "cluster", metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	// TODO(4.7): switch over to secure access-token logging by default and delete old non-sha256 tokens
```

Comment:
@sttts maybe we should create a BZ for this and set target release to 4.7 so we can track it?

Reply:
@sttts I created this BZ to track this: https://bugzilla.redhat.com/show_bug.cgi?id=1879182

Reply:
:+1: